robocopkaka/bootcamp-29-project
# #158334161 handle long event and center names

#### What does this PR do?
handle long event and center names
#### Description of Task to be completed?
* truncate event names in cards when they get too long
* truncate center names in cards when they get too long
* truncate center addresses in cards when they get too long
* properly display long event and center names in their details' page
#### How should this be manually tested?
* clone this repo
* `cd` into the project folder
* run `npm install` to install all dependecies
* run `npm install -g sequelize-cli` to install `sequelize` globally
* run `npm run start:all` to start up the application
* go to `localhost:8000` on your browser
* click on `Signup`
* enter your details
* click on `Signup`
* request for admin access
* click on `Centers` on the navbar
* click on the red add button at the bottom right corner
* enter the details of the center with a really long name
* click on `Add Center`
* you should see a card representing the center with its name truncated at some point
* add at least ten more centers
* click on Centers
* select the center with the long name
* you should see the name fit within its page
* click on the red add button at the bottom right corner
* enter the details of the event with a really long name
* click on `Add Event`
* you should see a card representing the event with its name truncated at some point
* click on the event
* you should see the name fit within its page
#### Any background context you want to provide?
n/a
#### What are the relevant pivotal tracker stories?(if applicable)
#158334161
